Corona-Warn-App/src/main/res/layout/fragment_information_terms.xml

```diff
@@ -10,75 +10,37 @@
 
     </data>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <LinearLayout
         android:id="@+id/information_terms_container"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:contentDescription="@string/information_terms_title"
-        android:focusable="true">
+        android:focusable="true"
+        android:orientation="vertical">
 
         <include
             android:id="@+id/information_terms_header"
             layout="@layout/include_header"
-            android:layout_width="0dp"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             app:icon="@{@drawable/ic_back}"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
             app:title="@{@string/information_terms_title}" />
 
         <ScrollView
-            android:layout_width="0dp"
-            android:layout_height="0dp"
-            android:fillViewport="true"
-            app:layout_constraintBottom_toBottomOf="@+id/guideline_bottom"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/information_terms_header">
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:fillViewport="true">
```

Comment:
You might be missing a `paddingBottom` here if you want the same behavior as previously.

```suggestion
            android:paddingBottom="@dimen/guideline_bottom"
            android:fillViewport="true">
```

Or, if you prefer:


```suggestion
            android:paddingBottom="@dimen/spacing_small"
            android:fillViewport="true">
```

Reply:
Regarding this padding (which the other layouts have, by the way): are you aware of the property `clipToPadding`? Setting it to false might be an effect that you desire, as the `ScrollView` would still have empty space below its last view but wouldn't clip the content to that padding anymore, allowing content to be rendered in those last 16dp of the screen as well.

Reply:
I did not check for that specifically, I just scanned the layout after refactoring to make sure it still looks good.

In terms of solutions i generally prefer `android:paddingBottom="@dimen/spacing_small"`, I don't like all the indirections with guidelines, includes, dimens we currently have, one screen refactoring at a time :wink: 

@ralfgehrer Do you spot the difference with padding here?

Reply:
Looks all the same with and without padding on my device. Hence, I'd leave it as is. Do you agree fynngodau?

Reply:
I was pretty sure that this change would have the same effect as the guideline that is being removed from lines 75-80 had. I'll have another look.

Reply:
I was correct that this change would make it uniform with all the other views; you are correct that it is not useful. See the attached screenshots (with layout bounds):

![tos-withoutpadding](https://user-images.githubusercontent.com/16943720/102242590-80e26980-3efa-11eb-9e19-062875fc445b.png)
![tos-withpadding](https://user-images.githubusercontent.com/16943720/102242596-82ac2d00-3efa-11eb-822d-3cfa82e63af1.png)

The padding adds the white space that clips the text at the bottom. All other screens have that clipping.

Like I mentioned, I suppose the intended effect is that text doesn't align to the bottom without any margin, though like I also said above, this could be achieved in such a way that it doesn't clip the text (which I recommend).

Of course, here, the text has enough empty lines at the bottom that it has margin to the bottom anyway.

You will have to decide whether you want to make it uniform to other views.

Reply:
I don't think the clipping with whitespace looks good.
I'll merge this now and I'd be open to follow up PRs that introduce uniform whitespace at the bottom without clipping text, e.g. some "overscroll".